code/game/machinery/recycler.dm

```diff
@@ -74,20 +71,9 @@ var/const/SAFETY_COOLDOWN = 100
 	add_fingerprint(user)
 	return
 
-/obj/machinery/recycler/emag_act(mob/user)

```

Comment:
You have REMOVED the emag_act, AND the safety mode! This means that it WILL ALWAYS CRUSH! Do you see why this is a bad idea, Goof!?!?!?!?!?!?!?!?!?!